src/api-interface.js

```diff
@@ -0,0 +1,25 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const gameId = 'Rcl2sFmCQZv9b27vyvUs';
+
+const targetUrl = `${apiUrl}games/${gameId}/scores`;
```

Comment:
While you saved a leg work for you to define manually the game ID 🤓 
it's a requirement to create a game using the API and use it to submit the scores. [LINK](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Create%20a%20new%20game%20with%20the%20name%20of%20your%20choice%20by%20using%20the%20API.)

Kindly make a new fetch request in `async/await` style to create a new game with the help of the API documentation. 📜